src/sentry/static/sentry/app/views/performance/transactionSummary/table.tsx

```diff
@@ -52,12 +52,12 @@ class SummaryContentTable extends React.Component<Props> {
   renderHeader() {
     const {eventView, tableData} = this.props;
 
-    const tableDataMeta = tableData && tableData.meta ? tableData.meta : undefined;
+    const tableMeta = tableData && tableData.meta ? tableData.meta : undefined;
```

Comment:
I suppose this isn't equivalent if `meta` is falsey and not undefined

```suggestion
    const tableMeta = tableData?.meta
```

Reply:
It is always present or undefined, so this is a safe transform.